service/keyv2/key.go

```diff
@@ -112,11 +115,21 @@ func MasterHostPathVolumeDir(clusterID string, vmNumber string) string {
 	return filepath.Join("/home/core/volumes", clusterID, "k8s-master-vm"+vmNumber)
 }
 
+// MemoryQuantity returns a resource.Quantity that represents the memory to be used by the nodes.
+// It adds the memory from the node definition parameter to the additional memory defined by
+// defaultWorkerMemory.
```

Comment:
I like that we are staring to do comments (requirement of `go lint` at least).

Reply:
Thanks! :)